### PR TITLE
Dec 2023 draft newsletter

### PR DESCRIPTION
**Pull request by ubdbra001**

### CODE REVIEW

File: _newsletters/2023-12-04-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+- Interesting piece on code complexity and software sustainability: [The sustainability doughnut of scientific software](https://science-in-the-digital-era.khinsen.net/#The%20sustainability%20doughnut%20of%20scientific%20software)
+
+- [The low-hanging fruit in computational reproducibility](http://blog.khinsen.net/posts/2023/11/30/The-low-hanging-fruit-in-computational-reproducibility.html)
+
+#### Publications
+
+- [Retracted articles use less free and open-source software and cite it worse](https://direct.mit.edu/qss/article/doi/10.1162/qss_a_00275/118074/Retracted-articles-use-less-free-and-open-source)
+
+- **Pre-Print** [A Comparative Study of Code Generation using ChatGPT 3.5 across 10 Programming Languages (pre-print)](https://arxiv.org/abs/2308.04477)
```
Comment (davidwyld) [inline _newsletters/2023-12-04-newsletter.md]:
```suggestion
- [A Comparative Study of Code Generation using ChatGPT 3.5 across 10 Programming Languages (pre-print)](https://arxiv.org/abs/2308.04477)
```

File: _newsletters/2023-12-04-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+- [The Luck of the Draw: Wellcome's Institutional Fund for Research Culture](https://wellcomeopenresearch.org/articles/8-525/v1)
+
+### Events
+
+- [DRA Train-the-Trainer program #3](https://events.digital-research.academy/event/32/) - Starting in January 2024, Applications are open until December 15th 2023.
```
Comment (davidwyld) [inline _newsletters/2023-12-04-newsletter.md]:
```suggestion
- [Digital Research Academy Train-the-Trainer program #3](https://events.digital-research.academy/event/32/) - Starting in January 2024, Applications are open until December 15th 2023.
```

**Review by davidwyld — APPROVED** [review]
Looking good! Thanks @ubdbra001 !

File: _newsletters/2023-12-04-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+- [Announcement for the location and theme of the SSI Collaboration Workshops 2024](https://www.software.ac.uk/news/announcing-theme-and-location-collaborations-workshop-2024) - 30 April to 02 May 2024 at the University of Warwick.  
+Themes encompass: Environmental Sustainability, AI/ML tools for science, and Citizen science
+
+- [43 UK institutions reforming recruitment and promotion](https://www.ukrn.org/2023/11/20/43-uk-institutions-reforming-recruitment-and-promotion/)
+
+- [NVIDIA Is A New Sponsor Of The Scikit-Learn consortium at the Inria Foundation](https://blog.scikit-learn.org/funding/nvidia-is-a-new-sponsor/)
+
+- PyPi has completed its first security audit, you can read about what was audited, what was found, and how it was remediated in three parts ([Part 1](https://blog.pypi.org/posts/2023-11-14-1-pypi-completes-first-security-audit/), [Part 2](https://blog.pypi.org/posts/2023-11-14-2-security-audit-remediation-warehouse/), [Part 3](https://blog.pypi.org/posts/2023-11-14-3-security-audit-remediation-cabotage/))
+- [Research Software Alliance newsletter](https://preview.mailerlite.com/c3q9q5c5p3/2354512809695386395/k7l2/)
```
Comment (ns-rse) [inline _newsletters/2023-12-04-newsletter.md]: Perhaps overkill...

```suggestion
- [Research Software Alliance newsletter (November 2023)](https://preview.mailerlite.com/c3q9q5c5p3/2354512809695386395/k7l2/)
```

**Review by ns-rse — APPROVED** [review]
Great work @ubdbra001 

One minor suggestion, not at all essential.

**Review by Robadob — APPROVED** [review]
Only had a quick look, but looks solid.
My usual complaint about inconsistent/missing dates isn't valid in this case!

File: _newsletters/2023-12-04-newsletter.md (hunk 3 above)
Comment (ubdbra001) [reply _newsletters/2023-12-04-newsletter.md]: Nah, I thought of it just after I put in the pull request.

File: _newsletters/2023-12-04-newsletter.md (hunk 1 above)
Comment (ubdbra001) [reply _newsletters/2023-12-04-newsletter.md]: Altered the text for the other pre-print too, for consistency.

**Pull request merged**